### PR TITLE
pi: increase packets' payload

### PR DESCRIPTION
**Pull request by masco-enovance**
increase the packets' payload on each packet

### CODE REVIEW

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-orientdb

**Comment by lebauce** [comment]
run skydive-functional-tests-backend-elasticsearch

**Comment by masco-enovance** [comment]
run skydive-k8s-tests

**Comment by masco-enovance** [comment]
run skydive-k8s-tests

**Comment by masco-enovance** [comment]
run skydive-k8s-tests

**Comment by masco-enovance** [comment]
run skydive-k8s-tests

**Comment by masco-enovance** [comment]
run skydive-k8s-tests

**Comment by masco-enovance** [comment]
rerun skydive-cdd-overview-tests
rerun skydive-functional-tests-backend-elasticsearch
rerun skydive-k8s-tests

File: cmd/client/packet_injector.go [hunk 1]
```diff
@@ -35,21 +35,22 @@ import (
...
+	dstIP           string
+	dstMAC          string
+	packetType      string
+	payload         string
+	pcap            string
+	id              int64
+	count           int64
+	interval        int64
+	increment       bool
+	incrementalLoad bool
```
[the remark this answers is not in the document]
Comment (masco-enovance) [reply cmd/client/packet_injector.go]: done

**Pull request merged**